tests/internal/scripts/ccuda/cupy_ravel_variable.py

```diff
@@ -0,0 +1,24 @@
+# pylint: disable=missing-function-docstring, disable=unused-variable, missing-module-docstring
+
+import cupy as cp
+from pyccel.decorators import kernel, types
+from pyccel import cuda
+
+@kernel
+@types('int[:]', 'int[:]', 'int[:]')
+def func(a, b, c):
+    i = cuda.threadIdx(0) + cuda.blockIdx(0) * cuda.blockDim(0)
+    print("Hello World! ", a[i], b[i], c[i])
+
+if __name__ == '__main__':
+    threads_per_block = 32
+    n_blocks = 1
+    c =[[1,2],[1,3]]
```

Comment:
Could you use tuples here as lists are not officially supported:
```suggestion
    c =((1,2),(1,3))
```

Reply:
Done in [3fb754f](https://github.com/pyccel/pyccel/pull/1222/commits/3fb754ffb4b4eb90b92fcb423f01a41c6843fa50).